### DISCUSSION TITLE
Snipping tool opens but can't take a screenshot as before and my notification center and action center don't work for a while after boot

### DISCUSSION THREAD

**Discussion by Pyrrhusn** [farag2/Sophia-Script-for-Windows#417] [Q&A] [answered]
Hi,
Expanding on the issues listed in the title; 
The shortcuts, printscr button, and snipping tool app work fine as the app opens and when I try a shortcut, the mouse cursor changes to 'loading' state but the real process of choosing an area and such doesn't work. Has this happened to someone else before? And how can I solve this issue as It's important because I make a lot of screenshots.

Regarding the quick settings/control center/notification area/action center issue, if I want to for example change my laptop's brightness and go to click on the area, the grey background opens but none of the icons are there, it's not interactive. Same thing happens If I go to click on the clock to see all my notifications, the click glass/border effect shows that the click worked and the notification items index also goes away, but the calendar and notification don't come out. Although after a long time of waiting, these fix themselves and work as intended. Really annoying, especially on a laptop. I would like your help on this as well.
Thanks.

(Using win 11, latest updates)

**Comment by farag2** [comment, marked as answer]
Hi. sorry for the late reply. :) I've never saw such bug. But I found thatut I found that I mixed up 2 strings with each other. I mixed up 2 strings with each other. Do not think, it's related with the bug, but try to fix it.

```powershell
New-ItemProperty -Path "HKCU:\Control Panel\Cursors" -Name Person -PropertyType ExpandString -Value "%SystemRoot%\Cursors\W11_dark_v2.2\person.cur" -Force
New-ItemProperty -Path "HKCU:\Control Panel\Cursors" -Name Pin -PropertyType ExpandString -Value "%SystemRoot%\Cursors\W11_dark_v2.2\pin.cur" -Force
```

Neither SophiApp, nor the script make any changes to the notification area. So I cannot help without seeing what's exactly going on with your system. if you prefer and it's needed I may have a look at your Windows remotely via AnyDesk. PM in Telegam: @sanctuary_d
